Lin_reg_updated_plate_notation

haikookhandor opened the pull request:
- [x] Linear Regression
- [x] Plate notations
- [x] Logistic Regression
Categorical distribution is already plotted in the distributions.ipynb file.

haikookhandor committed:
Lin_reg_updated_plate_notation

haikookhandor committed:
log_reg_updated

nipunbatra commented:
- Slide 18 is very busy. Need to make it less busier. Reduce content and move to next set of slides.
- Slide 20, Bernoulli is written as "b"ernoulli
- Slide 24 is not clear. We want P(D|theta). So, we first must discuss what is D. In this slide, have some D1 = {... } and then show the P(D|theta) as a function of theta. In next set of 2-3 slides, vary the dataset and make the same plot. This should correspondingly have a notebook with an interactive widget.
- Slide 41. Need to show the figure as an "animation". First show only the true function, then show observation. Then show the normal distribution.
- Slide 42. Build the graphical model in more detail. We need to show f(x) = x^Ttheta. We also need to show the Normal distribution (similar to how you have done the graphical model for normal distribution) 
- Slide 43 and in general. Have \pause and incrementally build (discuss with Sarth)
- Slide 43. Use ~ carefully. In some contexts is means similar and in some distributed as. But, you are saying they're equivalent. That has a different maths symbol. 
- Slide 43. Missing negative sign in last equation.
- Slide 44. It is not log likelihood but negative log likelihood
- Slide 44. Put an equation number and tcolorbox on the NLL 
- Slide 44. tcolorbox has written w instead of theta. 



Similar comments for logistic regression part.

haikookhandor committed:
changes_made

haikookhandor commented:
I have addressed the comments. For animated gifs, I have stored it in the folder notebooks itself. For Bernoulli distribution slider image, it is updated in the notebook mle_bernoulli.ipynb in notebooks folder.

haikookhandor committed:
added_sliders_lin_log_reg

haikookhandor commented:
Sir I have made the changes.
